SPECS/kernel/config_aarch64

```diff
@@ -2321,6 +2321,7 @@ CONFIG_PROC_EVENTS=y
 #
 CONFIG_ARM_SCMI_PROTOCOL=m
 # CONFIG_ARM_SCMI_RAW_MODE_SUPPORT is not set
+# CONFIG_ARM_SCMI_DEBUG_COUNTERS is not set
```

Comment:
result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/0b175f7c4fe9dee65e1b5164bc87bb6c729064ae